uber/models.py

```diff
@@ -133,20 +128,22 @@ def process_bind_param(self, value, dialect):
         return value if isinstance(value, str) else ','.join(value)
 
 
-@declarative_base
+default_metadata = MetaData(
+    naming_convention=immutabledict({
+        'unnamed_ck': check_constraint_naming_convention,
+        'ix': 'ix_%(column_0_label)s',
+        'uq': 'uq_%(table_name)s_%(column_0_name)s',
+        'ck': 'ck_%(table_name)s_%(unnamed_ck)s',
+        'fk': 'fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s',
+        'pk': 'pk_%(table_name)s'}))
```

Comment:
There is a whole section of the alembic docs on this very issue: http://alembic.zzzcomputing.com/en/latest/naming.html

But your point about adding an explanatory comment is a good one.